src/NCronJob/Registry/JobRegistry.cs

```diff
@@ -66,14 +66,14 @@ public JobDefinition AddDynamicJob(
 
         var entry = JobDefinition.CreateUntyped(DynamicJobNameGenerator.GenerateJobName(jobDelegate), jobPolicyMetadata) with
         {
-            CustomName = jobName
+            CustomName = jobName,
         };
 
         if (jobOption is not null)
         {
             Debug.Assert(jobOption.CronExpression is not null);
 
-            var cron = NCronJobOptionBuilder.GetCronExpression(jobOption.CronExpression);
+            var cron = NCronJobOptionBuilder.GetCronExpression(jobOption.CronExpression.Trim());
             entry.CronExpression = cron;
```

Comment:
How about centralizing the trimming in the setter of the JobDefinition.CronExpression property?